guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,254 @@
+    - file: wallet-backup/app-home-backup-done
+      alt: Image of the home screen that does not show the backup reminder anymore.
+      caption: The app home screen does not show the backup reminder anymore.
+---
+
+<!--
+
+Editor's notes
+
+This page covers how users backup their wallet and an example approach of how to store the backup material.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Inheritance-wallet?type=design&node-id=6293%3A21917&mode=design&t=I2e3qgqYRGpAGyaQ-1 
+
+-->
+
+# Wallet backup 
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+
+With the family savings wallet in place, establishing a robust backup strategy is crucial. However, there's no universal solution for wallet backups. Each family must tailor their approach based on their unique circumstances, considering factors such as:
+
+- Relationships between family members
+- Amount of funds stored
+- Access to secure storage locations
+- Technical comfort level of family members
+- Desired balance between security and accessibility
+
+The backup scheme outlined below is one potential approach. It aims to balance security, privacy, and ease of use, but may not suit every family's needs. As you read through this guide, consider how you might adapt these strategies to best fit your family's situation.
+
+## What needs to be backed up
+There are two parts to backing up a multi-key wallet: the private keys as well as the wallet configuration.
+
+- **Private Keys**: Secure backups of all private keys used in the wallet.
+- **Wallet Descriptor**: A small piece of text containing the wallet configuration.
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/multikey-backup-components.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/multikey-backup-components@2x.png"
+   alt-text = "An illustration showing the two parts of a multi-key wallet backup."
+   width = 560
+   height = 480
+%}
+
+#### Private keys
+Firstly, users need to back up the private keys. In our example, the Joneses need to safely backup and store six private keys. We describe some best practices on how to back up and store private keys on the [bitcoin backups page]({{ '/guide/how-it-works/backups/' | relative_url }}). 
+
+#### Wallet configuration
+In addition to the private keys, it is also necessary to back up the wallet configuration as such. This is needed to recover a multi-key wallet, because the wallet application needs to know how to generate bitcoin addresses as well as the rules that define how bitcoin can be spent from these addresses. The wallet configuration, also called the [wallet descriptor]({{ '/guide/glossary/#output-script-descriptor' | relative_url }}), is a small piece of text that contains all the information that a wallet application needs to recover a multi-key wallet. 
+
+#### Don’t store them together
```

Comment:
This could be presented with a custom tip box as described in the Formatting guide.

{% include tip/open.html color="red" icon="forbid" label="Don't store private keys and wallet descriptor together" %}

If a third party gains access to the descriptor, they can import the wallet to their own application and monitor all past and future activity. However, they cannot spend any funds, as long as they don’t have access to the necessary number of private keys to sign transactions. This is why it is critical that private keys are stored separately from the wallet descriptor.

{% include tip/close.html %}